doc/sources/conf.py

```diff
@@ -68,8 +68,23 @@
     "sphinx_design",
     "sphinx_copybutton",
     "sphinx.ext.napoleon",
+    "sphinx.ext.intersphinx",
 ]
 
+intersphinx_mapping = {
```

Comment:
Wishful thinking: is there any way to get this from sklearn itself? I imagine this was a maintenance burden we just didn't track until your good work.

Reply:
I don't think so, because this is only part of the docs `conf.py` file, which is not part of the python package.